test/tests/functional/pbs_multi_sched.py

```diff
@@ -715,7 +715,13 @@ def test_list_multi_sched(self):
 
         self.server.manager(MGR_CMD_LIST, SCHED, id="sc1")
 
-        self.server.manager(MGR_CMD_LIST, SCHED, id="sc2")
+        dir_path = '/var/spool/pbs/sched_dir'
+        a = {'partition': 'P2',
+             'sched_priv': os.path.join(dir_path, 'sched_priv_sc2'),
+             'sched_log': os.path.join(dir_path, 'sched_logs_sc2'),
+             'sched_host': self.server.hostname,
+             'sched_port': '15051'}
+        self.server.manager(MGR_CMD_LIST, SCHED, a, id="sc2", expect=True)
```

Comment:
Don't use expect=True on a MGR_CMD_LIST

Reply:
Ok removed expect=True

Reply:
It's a little confusing, but expect=True with a list of attributes would actually verify that qmgr list returned the correct attribute values. If you don't wanna depend on expect=True, then you could also explicitly check that scheduler.attributes was populated correctly. Up to you, but your code was previously correct.

Reply:
Ok I put back the expect=True